antidox/tester.py

```diff
@@ -0,0 +1,79 @@
+""" Program that's gets the latest revisions and checks for PII and toxicity """
```

Comment:
Please modify this docstring to mention how this program is different from `wikiwatcher.py`. Mention that this works on custom/local Mediawiki installations since it doesn't use `seeclient`. Also mention that you need to have a custom `doxwiki_family.py` file.

Reply:
I mentioned it in the readme file, but you're saying I also need to emphasize the dox family thing in the actual code?